go.mod

```diff
@@ -70,6 +71,7 @@ require (
 	github.com/spf13/viper v1.19.0
 	github.com/stretchr/testify v1.9.0
 	github.com/tsg/go-daemon v0.0.0-20200207173439-e704b93fd89b
+	github.com/xuri/excelize/v2 v2.8.1
```

Comment:
why do we need `excelize` in the main go.mod if you created a separated go dependency context in `scripts/update_assets_md`?

Reply:
Same about `strcase`

Reply:
Moved dependencies to a separate `go.mod/sum` 👍